SYCL/SubGroup/info.cpp

```diff
@@ -40,7 +37,7 @@ int main() {
     auto Kernel = KB.get_kernel(KernelID);
     range<2> GlobalRange{50, 40};
 
-    buffer<double, 2> ABuf{GlobalRange}, BBuf{GlobalRange}, CBuf{GlobalRange};
+    buffer<float, 2> ABuf{GlobalRange}, BBuf{GlobalRange}, CBuf{GlobalRange};
```

Comment:
I'm also touching it in https://github.com/intel/llvm-test-suite/pull/1246, but I prefer your change. I'll exclude the file from my PR.